### PR TITLE
john --> master: updated sharx config and added Samsung NVR

### PR DESCRIPTION
**Pull request by jhallard**

### CODE REVIEW

File: configs/README.MD [hunk 1]
```diff
@@ -15,7 +15,7 @@ The [configs](../master/configs/configs.csv) file contains a list of configurati
 | regex_firmware	| A regex representing all the firmware versions that support the comamnd, e.g "^v4" | required |
 | regex_success	| A regex that represents the response of a command successfully executed | optional |
 | auth_type	| The supported authentication option, e.g. “basic” or “digest” | optional |
-| payload | The name of a file in this directory that contains the payload to send to the given URL | optional |
+| payload   | name of the file in the `rtsp/configs/` directory that we will POST/PUT to the given url | optional |
```
Comment (jcmaslan) [inline configs/README.MD]: ...that will be used as the payload template for the POST/PUT to the given url.

File: configs/configs.csv [hunk 2]
```diff
@@ -1,5 +1,6 @@
 maker,command_id,url,method,auth_type,regex_success,regex_firmware,regex_model,payload
 Sharx Security,enable_wifi,http://{{ip_address}}/form/setWlan/?ENABLE=1&SSID={{wifi_ssid}}&MODE=INFRA&SECURITY=WPA2-PSK&ENCRYPTION=AES&AUTHTYPE=AUTO&KEYTYPE=HEX&WEPIDX=1&WEPKEY=&WPAKEY={{wifi_password}},GET,basic,^000 Success,^v5,.*SCNC3[89]0[45].*,
+Sharx Security,set_stream_{{stream}},http://{{ip_address}}//form/setStream/?TYPE=0&ENABLE=1&SIZE=HDSIZE&FRAMERATE=20&H264BITRATE=1536&MPEG4BITRATE=1024&MJPEGQUALITY=50&JPEGQUALITY=90&AUDIOCODEC=AACLC&AUDIOBITRATE=16000&AUTH=1&H264BUFFERSEC=0,GET,basic,^000 Success,^v5,.*SCNC3[89]0[45].*,
```
Comment (jcmaslan) [inline configs/configs.csv]: `set_stream_0` as the command_id

File: configs/README.MD (hunk 1 above)
Comment (jhallard) [reply configs/README.MD]: done with 8e08c8e72b1713199b91373b19f4c83d1fb5fc1c

File: configs/configs.csv (hunk 2 above)
Comment (jhallard) [reply configs/configs.csv]: done

**Comment by jcmaslan** [comment]
LGTM

**Pull request merged**